mindee/client.py

```diff
@@ -132,6 +138,8 @@ def enqueue(
         :param cropper: Whether to include cropper results for each page.
             This performs a cropping operation on the server and will increase response time.
         """
+        if self.input_doc is None:
+            raise RuntimeError("The 'enqueue' function requires an input document.")
```

Comment:
Maybe use `TypeError` instead?

Because `None` is of type `NoneType`, which is of a different type than `XxxInputSource`...

https://docs.python.org/3/library/exceptions.html#TypeError